### PR TITLE
Add recipe to dilute concentrated peroxide

### PR DESCRIPTION
**Pull request by pisskop**
A reverse of the above recipe, peroxide to concentrated peroxide.

provides 5 peroxides from 1 concentrated peroxide unit.

### CODE REVIEW

**Comment by Coolthulhu** [comment]
Why so hard? Isn't it just adding water to concentrated peroxide, which should be autolearn 0 difficulty?

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -2479,6 +2479,28 @@
...
   "type" : "recipe",
+  "result": "chem_hydrogen_peroxide",
+  "byproducts": { "id" : "chem_hydrogen_peroxide", "charges_mult" : 4 },
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_DRUGS",
+  "skill_used": "cooking",
+  "skills_required": [["firstaid", 3]],
+  "difficulty": 4,
+  "time": 6000,
+  "reversible": false,
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: `"reversible": false` is redundant.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 2]
```diff
@@ -2479,6 +2479,28 @@
...
+  "autolearn": true,
+  "book_learn": [["recipe_labchem", 3 ], ["textbook_chemistry", 3]],
+  "qualities":[
+  {"id":"BOIL","level":2,"amount":1},
+  {"id":"CHEM","level":1,"amount":1}
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: `"amount":1` is redundant

Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: Also, indentation: should get 2 or 4 extra spaces

**Comment by pisskop** [comment]
Its hard for the reverse, which while including boiling up pseudo rocket fuel, this one is also a little OP.

Go on a lab raid, snatch up 6 bottles of (3) concentrated peroxide, and then make 90

File: data/json/recipes/recipe_medsandchemicals.json [hunk 3]
```diff
@@ -2479,6 +2479,28 @@
 },
 {
   "type" : "recipe",
+  "result": "chem_hydrogen_peroxide",
+  "byproducts": { "id" : "chem_hydrogen_peroxide", "charges_mult" : 4 },
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: Using byproducts for that is not the proper way. Should be:
```
"result_mult": 5
```

**Comment by Rivet-the-Zombie** [comment]
>Why so hard? Isn't it just adding water to concentrated peroxide, which should be autolearn 0 difficulty?

The water needs to be pure distilled, else the peroxide will begin reacting with whatever contaminants are in there.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 4]
```diff
@@ -2478,6 +2478,30 @@
...
+    "time": 6000,
+    "autolearn": true,
+    "book_learn": [["recipe_labchem", 3 ], ["textbook_chemistry", 3]],
+    "qualities":[
+      {"id":"BOIL","level":2},
+      {"id":"CHEM","level":1}
+    ],
+    "tools": [
+    [[ "hotplate", 10 ], [ "toolset", 10 ], [ "fire", -1 ]]
+    ],
```
Comment (Coolthulhu) [inline data/json/recipes/recipe_medsandchemicals.json]: The whole `tools` line can be just a single line. Same for `qualities` above - breaking lines makes sense when they're too long, but when they're short like the `book_learn` above, it's better to keep them on one line.

**Pull request merged**